core_lib/src/structure/layercamera.cpp

```diff
@@ -19,6 +19,9 @@ GNU General Public License for more details.
 #include <QSettings>
 #include "camera.h"
 #include "pencildef.h"
+#include "cameraeasingtype.h"
+#include <QEasingCurve>
```

Comment:
```suggestion
#include <QSettings>
#include <QEasingCurve>

#include "camera.h"
#include "pencildef.h"
#include "cameraeasingtype.h"
```

Please keep avoid mixing Qt imports with our own headers